examples/stm32h7/src/bin/spi_bdma.rs

```diff
@@ -10,18 +10,24 @@ use embassy_executor::Executor;
 use embassy_stm32::mode::Async;
 use embassy_stm32::time::mhz;
 use embassy_stm32::{spi, Config};
+use grounded::uninit::GroundedArrayCell;
 use heapless::String;
 use static_cell::StaticCell;
 use {defmt_rtt as _, panic_probe as _};
 
 // Defined in memory.x
 #[link_section = ".ram_d3"]
-static mut RAM_D3: [u8; 64 * 1024] = [0u8; 64 * 1024];
+static mut RAM_D3: GroundedArrayCell<u8, { 64 * 1024 }> = GroundedArrayCell::uninit();
```

Comment:
Could this be just 256B since you only take 2x128 byte slices?

Reply:
I also think that.
I will fix it.

Reply:
It's now 256B :)